WireUI/Sources/WireSettingsUI/Account/BackupImportExport/Export/CreatingBackupProgressView/CreatingBackupProgressViewController.swift

```diff
@@ -67,7 +67,6 @@ final class CreatingBackupProgressViewController: UIViewController {
         descriptionLabel.textColor = BaseColorPalette.Grays.gray70
         descriptionLabel.adjustsFontForContentSizeCategory = true
         descriptionLabel.accessibilityIdentifier = "descriptionLabel"
-        descriptionLabel.accessibilityLabel = "" // TODO: [WPB-15466] fix accessibility
```

Comment:
Since there are no mentions in the Figma file, I assume the accessibility label is derived from the title.